### ISSUE TITLE
Recommended Quality Assurance scans/analysis after scanner ramp down? 

### ISSUE THREAD

**Issue by TerynWilkes** [CMRR-C2P/MB#285]
Hello, we had a major component in our Siemens Prisma-Fit fail last week, and the extended repair required our magnet to be ramped down, have both the body and gradient coils replaced, re-shimmed, and ramped back up. The majority of our researchers use the CMRR MB sequences and asked me to reach out for your recommendations on what types of QA analysis we should run after we are back up. The majority of our current studies are longitudinal, and/or have multiple scan sessions for each participant with some sessions being before the ramp down.

Thank you for any help/suggestions.

**Comment by pvelasco** [comment]
Just my 2 cents...:
We had a similar situation at the NYU's Center for Brain Imaging a year ago (although we didn't need to re-shim).
We collect daily QA runs using the CMRR sequence (GRE-EPI) on the phantom bottle with the typical acquisition parameters. Then compute different SNR and tSNR metrics.
We compared results before and after the fix and checked that they were comparable (new metrics were within the typical distribution we had before the repair).

**Comment by jpyles** [comment]
@TerynWilkes I agree with @pvelasco suggestions here. Do you have pre-failure QA scans on phantoms? 

If you don't, one thing you could look at is the Siemens QA that is run during the annual PM. That should give you some comparison metrics, especially the "long terms stability" scan metrics. If you have a regular participant that you have resting state or some standard fMRI scan from pre-failure, you could also bring that person back and rescan them and compare tSNR, distortion, etc. Basically just run both datasets through [MRIQC](https://mriqc.readthedocs.io/en/stable/). Ideally you'd bring that person back a third time, then you would know variability of your measurements from session to session with no hardware changes, and then you can see if the variability before and after the hardware change is any different. 

I suspect you'll be fine and there shouldn't be any issues with before and after the gradient and body coil replacement. I would personally be more concerned about doing regular quality checks on your head coil to make sure all your elements are working and stable (we run coil check every day as part of daily qa). 

You might also want to ask this question over on [www.snug-discuss.org](https://www.snug-discuss.org/). 

Good luck! 

--John
